### ISSUE TITLE
[BUG] conflicting IDs when including same sls file both absolute and relative

### ISSUE THREAD

**Issue by sblaisot** [saltstack/salt#56720]
**Description**

When including the same sls file both absolute and relatives you get conflicting IDs, even if they come from the same file.

**Setup**

`/srv/salt/top.sls`:

```
base:
  '*':
    - testinclude
    - testinclude.subsls
```

`/srv/salt/testinclude/init.sls`:

```
/tmp/salt_test:
  file.managed:
    - user: root
    - group: root
    - mode: "0644"
    - contents: "salt test"
```

`/srv/salt/testinclude/subsls.sls`:

```
include:
  - .init
```

**Steps to Reproduce the behavior**

1. add the above three files
2. apply highstate on a minion
3. see conflicting ID between `base:testinclude` and `base:testinclude.init`, which are in fact the same file.

<details>
<summary>**relevant debug log** (click to expand)</summary>

```
[DEBUG   ] Could not find file 'salt://testinclude.sls' in saltenv 'base'
[DEBUG   ] In saltenv 'base', looking at rel_path 'testinclude/init.sls' to resolve 'salt://testinclude/init.sls'
[DEBUG   ] In saltenv 'base', ** considering ** path '/var/cache/salt/minion/files/base/testinclude/init.sls' to resolve 'salt://testinclude/init.sls'
[DEBUG   ] compile template: /var/cache/salt/minion/files/base/testinclude/init.sls
[DEBUG   ] Jinja search path: ['/var/cache/salt/minion/files/base']
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/init.sls' using 'jinja' renderer: 0.001132965087890625
[DEBUG   ] Rendered data from file: /var/cache/salt/minion/files/base/testinclude/init.sls:
/tmp/salt_test:
  file.managed:
    - user: root
    - group: root
    - mode: "0644"
    - contents: "salt test"

[DEBUG   ] Results of YAML rendering: 
OrderedDict([('/tmp/salt_test', OrderedDict([('file.managed', [OrderedDict([('user', 'root')]), OrderedDict([('group', 'root')]), OrderedDict([('mode', '0644')]), OrderedDict([('contents', 'salt test')])])]))])
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/init.sls' using 'yaml' renderer: 0.0005359649658203125
[DEBUG   ] In saltenv 'base', looking at rel_path 'testinclude/subsls.sls' to resolve 'salt://testinclude/subsls.sls'
[DEBUG   ] In saltenv 'base', ** considering ** path '/var/cache/salt/minion/files/base/testinclude/subsls.sls' to resolve 'salt://testinclude/subsls.sls'
[DEBUG   ] compile template: /var/cache/salt/minion/files/base/testinclude/subsls.sls
[DEBUG   ] Jinja search path: ['/var/cache/salt/minion/files/base']
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/subsls.sls' using 'jinja' renderer: 0.0012164115905761719
[DEBUG   ] Rendered data from file: /var/cache/salt/minion/files/base/testinclude/subsls.sls:
include:
  - .init

[DEBUG   ] Results of YAML rendering: 
OrderedDict([('include', ['.init'])])
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/subsls.sls' using 'yaml' renderer: 0.0003287792205810547
[DEBUG   ] In saltenv 'base', looking at rel_path 'testinclude/init.sls' to resolve 'salt://testinclude/init.sls'
[DEBUG   ] In saltenv 'base', ** considering ** path '/var/cache/salt/minion/files/base/testinclude/init.sls' to resolve 'salt://testinclude/init.sls'
[DEBUG   ] compile template: /var/cache/salt/minion/files/base/testinclude/init.sls
[DEBUG   ] Jinja search path: ['/var/cache/salt/minion/files/base']
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/init.sls' using 'jinja' renderer: 0.0010983943939208984
[DEBUG   ] Rendered data from file: /var/cache/salt/minion/files/base/testinclude/init.sls:
/tmp/salt_test:
  file.managed:
    - user: root
    - group: root
    - mode: "0644"
    - contents: "salt test"

[DEBUG   ] Results of YAML rendering: 
OrderedDict([('/tmp/salt_test', OrderedDict([('file.managed', [OrderedDict([('user', 'root')]), OrderedDict([('group', 'root')]), OrderedDict([('mode', '0644')]), OrderedDict([('contents', 'salt test')])])]))])
[PROFILE ] Time (in seconds) to render '/var/cache/salt/minion/files/base/testinclude/init.sls' using 'yaml' renderer: 0.0004985332489013672
```
</details>  

**Expected behavior**

File `/tmp/salt_test` is created on minion

**Versions Report**
<details><summary>salt --versions-report</summary>

```
# salt-call -V
Salt Version:
           Salt: 2019.2.3
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.5.3
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.9.4
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.5.3 (default, Sep 27 2018, 17:25:39)
   python-gnupg: Not Installed
         PyYAML: 3.12
          PyZMQ: 16.0.2
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.4.3
            ZMQ: 4.2.1
 
System Versions:
           dist: debian 9.11 
         locale: UTF-8
        machine: x86_64
        release: 4.9.0-11-amd64
         system: Linux
        version: debian 9.11 
```
</details>

**Comment by sblaisot** [comment]
Similar to #54179 so I can confirm this issue is NOT fixed as of 2019.2.3

**Comment by alexey-zhukovin** [comment]
@sblaisot thank you. 
duplicate of #54179 closing
